src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource_getOffenderNsisByCrn.java

```diff
@@ -52,6 +52,7 @@ public void getNsiByCrnAndNsiId() {
         assertThat(nsi.getExpectedStartDate()).isEqualTo(LocalDate.of(2019, Month.SEPTEMBER, 20));
         assertThat(nsi.getActualStartDate()).isEqualTo(LocalDate.of(2019, Month.OCTOBER, 2));
         assertThat(nsi.getLength()).isEqualTo(20L);
+        assertThat(nsi.getNotes()).isNull();
```

Comment:
it might have been nice to change the seed data and put note in, but it such a low risk change I'm sure it will work :-)

Reply:
I'll do it.